src/app/personside/infotabs/meldinger/MeldingerContainer.tsx

```diff
@@ -51,6 +52,11 @@ function MeldingerContainer(props: MeldingerDyplenkeRouteComponentProps) {
     const traadIUrl = useValgtTraadIUrl(props);
     const [sokeord, setSokeord] = useState('');
 
+    const traaderEtterSok = useSokEtterMeldinger(hasData(traaderResource) ? traaderResource.data : [], sokeord);
+    if (traadIUrl && traaderEtterSok.length > 0 && !traaderEtterSok.includes(traadIUrl)) {
+        props.history.push(dyplenker.meldinger.link(traaderEtterSok[0]));
```

Comment:
Dette er en side-effect i render-cycle til react. 
Bruk `useEffect` eller `<Redirect to={path} />` for å endre url